BedrockServer.h

```diff
@@ -206,10 +206,18 @@ class BedrockServer : public SQLiteServer {
     // When a peer node logs in, we'll send it our crash command list.
     void onNodeLogin(SQLiteNode::Peer* peer);
 
-    // Control the command port. The server will toggle this as necessary, unless manualOverride is set,
-    // in which case the `suppress` setting will be forced.
+    // You must block and unblock the command port with *identical strings*.
```

Comment:
Since we have a fixed list of reasons, would it be worth making constants or an enum and verifying that the reason provided matches one of the accepted reasons? 

**Edit:** Maybe not, since we could have any number of `LEGACY_` calls from other plugins

Reply:
Yeah, `auth` or other plugins can specify whatever they want, or I would have done this.